### PR TITLE
brilliant-cv:2.0.1

### PR DESCRIPTION
**Pull request by mintyfrankie**
I am submitting
- [ ] a new package
- [X] an update for a package

Description: Explain what the package does and why it's useful.

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name that isn't the most obvious or canonical name for what the package does
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] tested my package locally on my system and it worked
- [X] `exclude`d PDFs or README images, if any, but not the LICENSE

- [X] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/brilliant-cv/2.0.1/template/cv.pdf
Comment (elegaanz) [inline packages/preview/brilliant-cv/2.0.1/template/cv.pdf:1]: Can you please delete this file?

**Comment by mintyfrankie** [comment]
Oups it got passed away! Just deleted, thanks for the notice!

A side question: since right now I have to manually copy/paste the folder locally from my repo to this repo, things like this can happen. Do we have any best practices (like having a CI/CD pipeline) that can automate this process and hence avoid human errors?

**Comment by elegaanz** [comment]
Thanks :) To answer your question: not that I know of. I think it is planned to add a `publish` command as part of [utpm](https://github.com/Thumuss/utpm) at some point, and I personally run [`typst-package-check`](https://github.com/typst/package-check/) to catch common mistakes such as this one (which you can probably run in CI), but I don't know of any better solution. Maybe ask on the Discord or the GitHub discussions of typst/typst, the community may use some tools that I am not aware of.

**Comment by fuchs-fabian** [comment]
> Oups it got passed away! Just deleted, thanks for the notice!
> 
> A side question: since right now I have to manually copy/paste the folder locally from my repo to this repo, things like this can happen. Do we have any best practices (like having a CI/CD pipeline) that can automate this process and hence avoid human errors?

@mintyfrankie 

I feel for you. I had and have the same problem (https://github.com/typst/packages/pull/847). I'm using a pipeline, but it's not able when in my typst template repo, once it's uploaded to the fork from typst packages once, to automatically pass the changes to the PR. After a little trial and error, I have 2 options that halfway work:

1. close PR and delete release + tag from own repo and delete the branch in the fork of typst packages. Then create a new release in your own repo and open the PR again
2. leave the PR open, change it and pull it into your own repository, with the result that the pipeline fails for you.

However, neither is optimal if changes have to be made through a PR :/

**Pull request merged**